assert/assertions_test.go

```diff
@@ -153,6 +153,21 @@ func TestObjectsAreEqual(t *testing.T) {
 
 }
 
+func TestObjectsAreEqual_Time(t *testing.T) {
```

Comment:
Should we not have the test name explicitly state that we're testing serialized/json time representations? i.e. `TestObjectsAreEqual_SerializedTime`?

Reply:
Done